From b44b3ed0363e138117f3b16565e4d8b79eeb76c8 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 12 Sep 2014 09:59:03 +0200
Subject: [PATCH] cms/cmssetup.sh: understand DASD="none"

Treat DASD="none" as unset.

https://bugzilla.redhat.com/show_bug.cgi?id=1096979
(cherry picked from commit 2f954621ccde0bc9160cd90142865e7586e25b00)
---
 modules.d/80cms/cmssetup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/80cms/cmssetup.sh b/modules.d/80cms/cmssetup.sh
index 7dd481b..6259ff4 100755
--- a/modules.d/80cms/cmssetup.sh
+++ b/modules.d/80cms/cmssetup.sh
@@ -175,7 +175,7 @@ processcmsfile()
         znet_cio_free
     fi
 
-    if [[ $DASD ]]; then
+    if [[ $DASD ]] && [[ $DASD != "none" ]]; then
         echo $DASD | normalize_dasd_arg > /etc/dasd.conf
         echo "options dasd_mod dasd=$DASD" > /etc/modprobe.d/dasd_mod.conf
         dasd_cio_free
